app/scripts/templates/sign_up.mustache

```diff
@@ -19,8 +19,13 @@
     {{^error}}
     <div class="error"></div>
     <div class="success"></div>
+
+    {{#showSyncSuggestion}}
+      <div class="info nudge" id="suggest-sync">{{#t}}Looking for Firefox Sync? <a href="https://mozilla.org/firefox/sync">Get started here.</a>{{/t}}</div>
+    {{/showSyncSuggestion}}
+    
     {{#isSyncMigration}}
-      <div class="info nudge">{{#t}}Migrate your sync data by creating a new Firefox&nbsp;Account.{{/t}}</div>
+      <div class="info nudge migration">{{#t}}Migrate your sync data by creating a new Firefox&nbsp;Account.{{/t}}</div>
```

Comment:
Actually looks like we had an `id` for amo-migration here: https://github.com/mozilla/fxa-content-server/blob/6120f1ebf7d1913b4976d935235e558edb8f9d97/app/scripts/templates/sign_up.mustache#L27

so you might choose ids over classes in this case